typescript/web/src/components/metabase/metabase-dashboard.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import { Flex } from "@chakra-ui/react";
+import { chakraDecorator } from "../../utils/stories";
+import { KpiDashboard } from "./kpi-dashboard";
+
+export default {
+  title: "web/Metabase Dashboard",
+  decorators: [chakraDecorator],
```

Comment:
As the rendered story will always have changing stats, Chromatic snapshots will always be considered as changed.
To fix this, we would have to create a metabase dashbord whose values never changes.
But instead, let's just disable the Chromatic snapshots for now 😉 

```suggestion
  decorators: [chakraDecorator],
  parameters: {
    chromatic: { disableSnapshot: true },
  },
```